src/orchestrator-process/src/lib.rs

```diff
@@ -214,6 +221,17 @@ impl NamespacedOrchestrator for NamespacedProcessOrchestrator {
                     pid_file_locations[i].as_ref().unwrap().display()
                 ));
 
+                // Pass down fields required to setup logging. We are
+                // running multi-process on one machine, so we assume
+                // collated log output.
+                if let Some(log_file) = &self.log_file {
+                    args.push(format!("--log-file={}", log_file));
```

Comment:
Not sure I'm reading this correctly, but are we letting multiple processes write to the same file? This .. will not work reliably in the limit. I'd favor having a specific log file for each process we start.

Reply:
Hmm, apparently this is supposed to work with POSIX-compliant file systems. I've been burnt too many times with corrupt logs because multithreaded programs, so I'm slightly biased. Also, it depends on the fs that's being written to.

Reply:
I was thinking it probably makes sense to ditch the `--log-file` argument entirely. It exists for the on premise world we were previously living in. In Docker world, you always write to stdout/stderr.

Reply:
I sent https://github.com/MaterializeInc/materialize/pull/12293 on this front.

Reply:
even easier!